web-components/section/section.js

```diff
@@ -0,0 +1,33 @@
+
+const template = document.createElement('template');
+
+template.innerHTML = `
+<style>
+ @import "web-components/section/section.css";
+</style>
+<div>
+  <div class="row">
+  <img src="/assets/img/dots.svg" ></img>
+    <div class="column">
+    <slot></slot>
```

Comment:
`slot` tag should have a name, it will be easier to keep track of the content we'll inject into it.